### PR TITLE
implement stub for query_chain

### LINKED ISSUE #321: Implement stub for query_chain

### PR DESCRIPTION
**Pull request by assafmo**
Closes #321

### CODE REVIEW

File: x/compute/internal/keeper/keeper_test.go [hunk 1]
```diff
@@ -305,7 +303,7 @@ func TestExecute(t *testing.T) {
 
 	addr, err := keeper.Instantiate(ctx, contractID, creator, nil, initMsgBz, "demo contract 3", deposit)
 	require.NoError(t, err)
-	require.Equal(t, "cosmos18vd8fpwxzck93qlwghaj6arh4p7c5n89uzcee5", addr.String())
+	require.Equal(t, "enigma18vd8fpwxzck93qlwghaj6arh4p7c5n89d2p9uk", addr.String())
```
Comment (reuvenpo) [inline x/compute/internal/keeper/keeper_test.go:306]: isn't this supposed to start with `secret1`?

File: x/compute/internal/keeper/keeper_test.go [hunk 2]
```diff
@@ -324,18 +322,20 @@ func TestExecute(t *testing.T) {
...
 
 	// verifier can execute, and get proper gas amount
 	start := time.Now()
 	gasBefore := ctx.GasMeter().GasConsumed()
 
-	res, err = keeper.Execute(ctx, addr, fred, msgBz, topUp)
+	msgBz, err := wasmCtx.Encrypt([]byte(`{"approve":{}}`))
+	require.NoError(t, err)
+
+	res, err := keeper.Execute(ctx, addr, fred, msgBz, topUp)
```
Comment (reuvenpo) [inline x/compute/internal/keeper/keeper_test.go:338]: I'm guessing that this test doesn't trigger the error you defined in this PR? At first I expected to see it checked here.

File: x/compute/internal/keeper/keeper_test.go (hunk 1 above)
Comment (assafmo) [reply x/compute/internal/keeper/keeper_test.go:306]: Not just yet... (#320)

File: x/compute/internal/keeper/keeper_test.go (hunk 2 above)
Comment (assafmo) [reply x/compute/internal/keeper/keeper_test.go:338]: Correct. It was crashing inside `keeper.Instantiate` with `Error in instantiation: Function("host module doesn\'t export function with name query_chain")`

**Comment by assafmo** [comment]
> Looks good :)
> wouldn't we prefer to return a null buffer to the contract instead of crashing it though?

That's what I did at first, but then I thought it's better to crash it than to silently fail.

**Pull request merged**